ember-cli-build.js

```diff
@@ -21,6 +21,7 @@ module.exports = function () {
   // relative to the project root and we want to use the tsconfig as-is.
   let tsTree = funnel('packages/@glimmer', {
     destDir: 'packages/@glimmer',
+    exclude: ['vm-babel-plugins'],
```

Comment:
Do we need to exclude? What breaks if we don't exclude?

Reply:
Ah, we don't actually need this, this was before I moved the package into the `build-utils` sub-directory. TL;DR is the whole `packages/@glimmer` directory is really tied into being compiled for client side execution. Build utils are just a different category, so I decided to move it out.